pkg/sql/catalog/systemschema/system.go

```diff
@@ -171,7 +171,9 @@ CREATE TABLE system.eventlog (
   "reportingID" INT8       NOT NULL,
   info          STRING,
   "uniqueID"    BYTES      DEFAULT uuid_v4(),
-  CONSTRAINT "primary" PRIMARY KEY (timestamp, "uniqueID")
+  event				  JSONB,
```

Comment:
What if we called this a `payload`?

Reply:
That would be fine, why do you prefer payload over event?

Reply:
Just for name chaining, `event_log.payload` rather than `event_log.event`. It's mostly an ergonomic preference

Reply:
Testing @angles-n-daemons